tests/input_files/poro_3D_psi_hex8_new_struc.dat

```diff
@@ -0,0 +1,961 @@
+NODE 184 COORD 5.0000000000000000e+00 -3.0000000000000000e+00 5.0000000000000000e+00
+NODE 185 COORD 5.0000000000000000e+00 -5.0000000000000000e+00 1.0000000000000000e+00
+NODE 186 COORD 5.0000000000000000e+00 -3.0000000000000000e+00 1.0000000000000000e+00
+NODE 187 COORD 5.0000000000000000e+00 -5.0000000000000000e+00 -1.0000000000000000e+00
+NODE 188 COORD 5.0000000000000000e+00 -3.0000000000000000e+00 -1.0000000000000000e+00
+NODE 189 COORD 5.0000000000000000e+00 -5.0000000000000000e+00 -3.0000000000000000e+00
+NODE 190 COORD 5.0000000000000000e+00 -3.0000000000000000e+00 -3.0000000000000000e+00
+NODE 191 COORD 5.0000000000000000e+00 -5.0000000000000000e+00 -5.0000000000000000e+00
+NODE 192 COORD 5.0000000000000000e+00 -3.0000000000000000e+00 -5.0000000000000000e+00
+NODE 193 COORD 5.0000000000000000e+00 -1.0000000000000004e+00 3.0000000000000000e+00
+NODE 194 COORD 5.0000000000000000e+00 -1.0000000000000000e+00 5.0000000000000000e+00
+NODE 195 COORD 5.0000000000000000e+00 -1.0000000000000000e+00 1.0000000000000000e+00
+NODE 196 COORD 5.0000000000000000e+00 -1.0000000000000000e+00 -1.0000000000000000e+00
+NODE 197 COORD 5.0000000000000000e+00 -1.0000000000000002e+00 -3.0000000000000000e+00
+NODE 198 COORD 5.0000000000000000e+00 -1.0000000000000000e+00 -5.0000000000000000e+00
+NODE 199 COORD 5.0000000000000000e+00 1.0000000000000004e+00 3.0000000000000000e+00
+NODE 200 COORD 5.0000000000000000e+00 1.0000000000000000e+00 5.0000000000000000e+00
+NODE 201 COORD 5.0000000000000000e+00 1.0000000000000000e+00 1.0000000000000000e+00
+NODE 202 COORD 5.0000000000000000e+00 1.0000000000000000e+00 -1.0000000000000000e+00
+NODE 203 COORD 5.0000000000000000e+00 1.0000000000000002e+00 -3.0000000000000000e+00
+NODE 204 COORD 5.0000000000000000e+00 1.0000000000000000e+00 -5.0000000000000000e+00
+NODE 205 COORD 5.0000000000000000e+00 3.0000000000000000e+00 2.9999999999999996e+00
+NODE 206 COORD 5.0000000000000000e+00 3.0000000000000000e+00 5.0000000000000000e+00
+NODE 207 COORD 5.0000000000000000e+00 3.0000000000000000e+00 1.0000000000000004e+00
+NODE 208 COORD 5.0000000000000000e+00 3.0000000000000000e+00 -1.0000000000000004e+00
+NODE 209 COORD 5.0000000000000000e+00 2.9999999999999996e+00 -2.9999999999999996e+00
+NODE 210 COORD 5.0000000000000000e+00 3.0000000000000000e+00 -5.0000000000000000e+00
+NODE 211 COORD 5.0000000000000000e+00 5.0000000000000000e+00 3.0000000000000000e+00
+NODE 212 COORD 5.0000000000000000e+00 5.0000000000000000e+00 5.0000000000000000e+00
+NODE 213 COORD 5.0000000000000000e+00 5.0000000000000000e+00 1.0000000000000000e+00
+NODE 214 COORD 5.0000000000000000e+00 5.0000000000000000e+00 -1.0000000000000000e+00
+NODE 215 COORD 5.0000000000000000e+00 5.0000000000000000e+00 -3.0000000000000000e+00
+NODE 216 COORD 5.0000000000000000e+00 5.0000000000000000e+00 -5.0000000000000000e+00
+------------------------------------------------STRUCTURE ELEMENTS
+1 SOLIDH8PORO HEX8 1 2 3 4 5 6 7 8 MAT 1 KINEM nonlinear EAS none
```

Comment:
What kind of poro formulation is used in this testcase? Can we use the new solid poro elements?

Reply:
Pressure velocity based. The new solid poro elements also works and is tested in the other 3D test. I chose to include the old elements as well since this test exists in the old time integration.

Reply:
I think the new solid poro elements also work for the old time integration, or am I understanding this wrong?

Reply:
Yes, both versions of the solid poro element work in the old time integration. The new test cases are just copies of the test cases in the old time integration with the main change being the use of the new time integration. The changes are independent of the used version of the solid element and I only included tests for both since both are tested in the old time integration. From my side we can also remove this test case if the migration to the new solid poro elements makes adding new tests with the old solid poro elements undesirable.

Reply:
Ah sorry, now I get it. My fault, thanks for the explanation. I think we want to get rid of the old solid elements at some point, so from my point of view, it would be sufficient to have only the test cases with the new solid elements. But I am also fine if you or @amgebauer have a different opinion on this.

Reply:
I removed the test with the old solid elements.